src/olympia/api/middleware.py

```diff
@@ -6,12 +6,15 @@
 
 
 class IdentifyAPIRequestMiddleware(MiddlewareMixin):
-    def process_request(self, request):
-        """Identify API requests.  Note this will not identify legacy API
-        requests - we can't do that reliably until after
-        LocaleAndAppURLMiddleware has activated."""
+    def identify_request(self, request):
         request.is_api = re.match(settings.DRF_API_REGEX, request.path_info)
 
+    def process_request(self, request):
+        self.identify_request(request)
+
+    def process_exception(self, request, exception):
+        self.identify_request(request)
+
```

Comment:
maybe not high priority here, but if we're rewriting IdentifyAPIRequestMiddleware we could drop the deprecated middleware style

Reply:
I don't want to make too many changes since I might cherry-pick this